numerical_methods/jacobi_method.cpp

```diff
@@ -0,0 +1,195 @@
+/**
+ * @file
+ * @brief Implementation of [The Jacobi Method]
+ (https://en.wikipedia.org/wiki/Jacobi_method#Algorithm)
+ * @details
+ * ### Algorithm
+ *
+ * 1) Input taken : A->(coefficient matrix), b->(column vector), ite->(number of
+ iterations)
+ * 2) Other variables taken : n->(order of A), prev->(initial column vector and
+ to store the current value),
+ * succ->(column vector to store updated value)
+ * 3) For (i=1, i<=ite, ++i){
```

Comment:
For codes like this, you can use the following formatting (this is just an example):

\```c++
for (int i = 0; i < 7; i++) {
    my_namespace::my_table[i] = i;
    std::cout << i << " ";
}
\```

This will display as

```c++
for (int i = 0; i < 7; i++) {
    my_namespace::my_table[i] = i;
    std::cout << i << " ";
}
```